_data/simple-icons.json

```diff
@@ -12426,7 +12426,13 @@
 	{
 		"title": "Plus Codes",
 		"hex": "4285F4",
-		"source": "https://maps.google.com/pluscodes/"
+		"source": "https://maps.google.com/pluscodes",
+		"guidelines": "https://maps.google.com/pluscodes/brand-guidelines",
+		"aliases": {
+			"aka": [
+				"Google Plus Codes"
+			]
+		}
```

Comment:
I can't find the name `Google Plus Codes` in the [brand guidelines](https://maps.google.com/pluscodes/brand-guidelines).

Let's remove the `aliases` part.